streaming-connect-sink/src/test/java/com/adobe/platform/streaming/integration/AbstractConnectorTest.java

```diff
@@ -139,6 +141,21 @@ public void inletJWTAuthenticationSuccessfulResponse() throws JsonProcessingExce
       .withJsonBody(JacksonFactory.OBJECT_MAPPER.readTree(AUTH_TOKEN_RESPONSE))));
   }
 
+  public void inletOAuth2IMSAuthenticationSuccessfulResponse() throws JsonProcessingException {
+    wiremockExtension.getWireMockServer()
+      .stubFor(WireMock
+      .post(WireMock.urlEqualTo(getRelativeOAuth2IMSAuthUrl()))
+      .willReturn(ResponseDefinitionBuilder.responseDefinition()
+      .withJsonBody(JacksonFactory.OBJECT_MAPPER.readTree(AUTH_TOKEN_RESPONSE))));
```

Comment:
AUTH_TOKEN_RESPONSE in case of OAuth will be different it won't contain refresh token.
Can you please add modified response ?